pyigm/guis/igmguesses.py

```diff
@@ -616,6 +617,7 @@ def __init__(self, ispec, z, parent=None, llist=None, norm=True, screen_scale=1.
         self.parent = parent
         self.spec = ispec
         self.spec_fil = self.spec.filename
+        self.orig_spec = ispec  # For smoothing
```

Comment:
You may need to use `.copy()` here

Reply:
And `.copy()` when you re-assign `self.spec` for un-smoothing.

Reply:
Thanks @profxj . Now using .copy(), in both cases. (Also, continuum-normalized orig_spec, and defined bad_pixels.)